test/test_cachefile_schema.py

```diff
@@ -3,18 +3,25 @@
 from copy import deepcopy
 
 from kernel_tuner.cache.json import CacheFileJSON, CacheLineJSON
-
+from kernel_tuner.cache.validate import validate_data
 import pytest
 import jsonschema
 import kernel_tuner
 
+from io import StringIO
+from contextlib import redirect_stdout
+import sys
```

Comment:
Omdat validate.py nu als een script is geschreven, heb je extra imports nodig.

Reply:
Overigens, pytest heeft ook al hiervoor functies ingebouwd, dus deze imports kunnen dus in ieder geval weggehaald worden.